.ci/update-github-io.sh

```diff
@@ -17,9 +17,3 @@ git checkout "checkstyle-$TARGET_VERSION"
 
 echo "Generating web site"
 mvn -e --no-transfer-progress site -Pno-validations -Dmaven.javadoc.skip=false
-
-echo "Clone by ssh only to avoid passwords on push ..."
```

Comment:
This script becomes weird.
Name is not matching to what it does.
Let's have two shell scripts.
Generate-website.sh
Push-website-to-github-io.sh